crates/kornia-io/src/stream/capture.rs

```diff
@@ -156,12 +172,61 @@ impl StreamCapture {
 
         let frame_buffer = FrameBuffer {
             buffer,
+            sample: sample.clone(),
             width,
             height,
         };
 
         Ok(frame_buffer)
     }
+
+    // Helper function to convert NV12 to RGB
+    fn nv12_to_rgb(
+        buffer: &[u8],
+        width: usize,
```

Comment:
```suggestion
        cols: usize,
```
to follow same convention as in the rest of library for rows/cols